apps/storefront/src/components/layout/B3Nav.tsx

```diff
@@ -148,16 +148,22 @@ export default function B3Nav({ closeSidebar }: B3NavProps) {
     const subsidiariesPermission = getSubsidiariesPermission(routes);
 
     if (selectCompanyHierarchyId) {
-      routes = routes.filter((route) =>
-        route?.subsidiariesCompanyKey
+      routes = routes.filter((route) => {
+        if (route?.subsidiariesCompanyKey === 'quickOrderPad') {
+          return true;
+        }
+        return route?.subsidiariesCompanyKey
```

Comment:
```suggestion
        if (route?.subsidiariesCompanyKey) {
            if (route.subsidiariesCompanyKey === 'quickOrderPad')
                return true;
            return subsidiariesPermission[route.subsidiariesCompanyKey];
        }
        return false;
```
💅 We can combine the checks for route and route.subsidiarycompany into one

Reply:
[question]

Similar to `subsidiariesPermission[route.subsidiariesCompanyKey]`, do we need to check for the order permissions to gate the `quickOrder` route?